backend/companion/getKcpToken.js

```diff
@@ -1,17 +1,20 @@
 export async function getKcpToken() {
   const tokenUrl = 'https://kymatest.accounts400.ondemand.com/oauth2/token';
   const grantType = 'client_credentials';
+
+  const localCredentials = getLocalCredentials();
+  const secretManagerCredentials = getSecretManagerCredentials();
+
   const clientId =
-    process.env.COMPANION_KCP_AUTH_CLIENT_ID ?? getLocalCredentials()?.clientId;
+    secretManagerCredentials?.clientId ?? localCredentials?.clientId;
   const clientSecret =
-    process.env.COMPANION_KCP_AUTH_CLIENT_SECRET ??
-    getLocalCredentials()?.clientSecret;
+    secretManagerCredentials?.clientSecret ?? localCredentials?.clientSecret;
```

Comment:
This approach will always throw errors about local credentials on the deployed version. I suggest making it like it was previously. This approach will not trigger `getLocalCredentials()` on the deployed version.